introduction.md

```diff
@@ -0,0 +1,31 @@
+Welcome to the first Turing Data Story.
```

Comment:
The content in this file is very similar to the opening paragraphs in the data story itself, and I haven't seen a link ot this file anywhere else. Is it an old draft that needs deleting? Or if you want to keep a version of this around so it can be read without loading the notebook, would it be better to move it to `stories/README.md` so that it can be viewed easily on GitHub?